clientd3d/game.c

```diff
@@ -15,6 +15,9 @@ player_info player;
 room_type current_room;
 BOOL dataValid = FALSE;
 
+// Default to group 5 (STATS_INVENTORY).
+BYTE active_group = 5; 
```

Comment:
Size isn't really a concern here so I'd just make this an int (change everywhere).